src/js/contentscripts/socialwidgets.js

```diff
@@ -118,6 +118,10 @@ function createReplacementButtonImage(tracker, trackerElem) {
   button.setAttribute("class", "privacyBadgerReplacementButton");
```

Comment:
We no longer need this class, right?

Reply:
I don't think so, but I'll double check to make sure. I originally left it in because I didn't want to be too liberal with stripping things out.